docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md

```diff
@@ -86,20 +103,34 @@ Consider increasing the disk space per node. By default, you are entitled to 50G
 
 For information on scaling clusters, refer to [Scale and configure clusters](../../cloud-clusters/configure-clusters/).
 
-#### Fix CPU alerts
+#### Fix nodes reporting as down alerts
 
-YugabyteDB Managed sends a notification when CPU use on any node in the cluster exceeds the threshold, as follows:
+YugabyteDB Managed sends a notification when the number of nodes down in the cluster exceeds the threshold, as follows:
 
-- Node CPU use exceeds 70% on average for at least 5 minutes (Warning).
-- Node CPU use exceeds 90% on average for at least 5 minutes (Severe).
+- Up to 34% of all nodes in the cluster are reporting as down (Info).
```

Comment:
"Down" means "failed" in this doc, right?
I'm asking because "down" doesn't read well in a sentence. I realize that's the term developers use, but it would be better to rephrase it.